pkg/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "pkg",
```

Comment:
Suggest using a library name that's a bit more descriptive. Perhaps setupGloudSDK?